flux/values-okteto.yaml

```diff
@@ -0,0 +1,116 @@
+    readonly: false
+    # Username to use as git committer
+    user: "Weave Flux"
+    # Email to use as git committer
+    email: "support@weave.works"
+    # If set, commits will be signed with this GPG key.
+    signingKey: ""
+    # If set, the signatures of the sync tag and commits will be verified.
+    verifySignatures: false
+    # If set, the author of git commits will reflect the user who initiated the commit and will differ from the git committer.
+    setAuthor: false
+    # Label to keep track of sync progress
+    label:
+    # Append "[ci skip]" to commit messages so that CI will skip builds
+    ciSkip: false
+    # Period at which to poll git repo for new commits
+    pollInterval: "1m"
+    # Duration after which git operations time out
+    timeout: "20s"
+    # Global Git configuration See https://git-scm.com/docs/git-config for more details.
+    config:
+      enabled: false
+      secretName: ""
+      data: ""
+      # data: |
+      #   [credential "https://github.com"]
+      #           username = foo
+
+  sync:
+    # use `.sync.state: secret` to store flux's state as an annotation on the secret (instead of a git tag)
+    state: git
+
+  gpgKeys:
+    # These keys will be imported into GPG in the Flux container.
+    secretName: ""
+    # These keys will be imported into GPG in the Flux container.
+    # NB: should only be used for public keys used to verify the
+    # signatures of commits.
+    configMapName: ""
+
+  # If `true` SOPS support will be enabled
+  sops:
+    enabled: false
+
+  registry:
+    # Period at which to check for updated images
+    automationInterval: "1m"
+    # Maximum registry requests per second per host
+    rps: 200
+    # Maximum number of warmer connections to remote and memcache
+    burst: 125
+    # Output trace of image registry requests to log
+    trace: false
+    # Disable registry scanning completely
+    disableScanning: false
+    # Do not scan images that match these glob expressions
+    excludeImage:
+
+  syncGarbageCollection:
+    enabled: false
+    dry: false
+
+  # Enables manifest generation
+  manifestGeneration: false
+
+helm-operator:
+  image:
+    repository: docker.io/fluxcd/helm-operator
+    tag: 1.0.0-rc9
+    pullPolicy: IfNotPresent
+    pullSecret:
+
+  # Log the diff when a chart release diverges
+  logReleaseDiffs: false
+  # Period on which to reconcile the Helm releases with `HelmRelease` resources
+  chartsSyncInterval: "1m"
+  # Amount of workers processing releases
+  workers: 2
+
+  # For charts stored in Git repos set the SSH private key secret
+  git:
+    # Period on which to poll git chart sources for changes
+    pollInterval: "5m"
+    timeout: "20s"
+    # Ref to clone chart from if ref is unspecified in a HelmRelease,
+    # empty defaults to `master`
+    defaultRef: ""
+    # Global Git configuration See https://git-scm.com/docs/git-config for more details.
+    ssh:
+      secretName: "${RELEASE_NAME}-flux-git-deploy"
```

Comment:
We call this `NAME` in Okteto Cloud, using `RELEASE_NAME` might be confusing no? Or should we change it in the ui to match this?